### DISCUSSION TITLE
is there any class in tailwind in parent element to apply same style in all child elements, like in below code, how can I give class in tr of tbody so repeatation i can avoid.

### DISCUSSION THREAD

**Discussion by jaimeenkoffeekodes** [tailwindlabs/tailwindcss#12947] [Help]
<table className="min-w-full divide-y divide-gray-200 dark:divide-gray-700">
                <thead>
                  <tr className="bg-[--green-blue] text-white">
                    <th
                      scope="col"
                      className="px-6 py-3 text-start text-xsmall font-medium text-white uppercase"
                    >
                      ID
                    </th>
                    <th
                      scope="col"
                      className="px-6 py-3 text-start text-xsmall font-medium text-white uppercase"
                    >
                      NAME
                    </th>
                    <th
                      scope="col"
                      className="px-6 py-3 text-start text-xsmall font-medium text-white uppercase"
                    >
                      CREATED
                    </th>
                    <th
                      scope="col"
                      className="px-6 py-3 text-start text-xsmall font-medium text-white uppercase"
                    >
                      CODE
                    </th>
                    <th
                      scope="col"
                      className="px-6 py-3 text-start text-xsmall font-medium text-white uppercase"
                    >
                      FROM-TO
                    </th>
                    <th
                      scope="col"
                      className="px-6 py-3 text-start text-xsmall font-medium text-white uppercase"
                    >
                      GET IN
                    </th>
                    <th
                      scope="col"
                      className="px-6 py-3 text-start text-xsmall font-medium text-white uppercase"
                    >
                      ADD GST
                    </th>
                    <th
                      scope="col"
                      className="px-6 py-3 text-start text-xsmall font-medium text-white uppercase"
                    >
                      ACTION
                    </th>
                  </tr>
                </thead>
                <tbody className="divide-y divide-gray-200 ">
                  {company_data.map((ele: company_data_interface) => (
                    <tr key={ele.ID}>
                      <td className="px-6 py-4 whitespace-nowrap text-small font-medium text-gray-800 ">
                        {ele.ID}
                      </td>
                      <td className="px-6 py-4 whitespace-nowrap text-small text-gray-800 cursor-pointer">
                        <Link href={`/${ele.ID}`}>{ele.Name}</Link>
                      </td>
                      <td className="px-6 py-4 whitespace-nowrap text-small text-gray-800 ">
                        {ele.Created}
                      </td>
                      <td className="px-6 py-4 whitespace-nowrap text-small text-gray-800 ">
                        {ele.FromTo}
                      </td>
                      <td className="px-6 py-4 whitespace-nowrap text-small text-[--blue-text]">
                        {ele.GetIn}
                      </td>
                      <td className="px-6 py-4 whitespace-nowrap text-small text-[--blue-text] ">
                        {ele.Code}
                      </td>
                      <td className="px-6 py-4 whitespace-nowrap text-small text-[--blue-text] ">
                        {ele.AddGst}
                      </td>
                      <td className="px-6 py-4 whitespace-nowrap text-end text-small font-medium">
                        <ActionButton id={ele.ID} />
                      </td>
                    </tr>
                  ))}
                </tbody>
              </table>`

**Comment by wongjn** [comment]
You could consider using the [direct children variant](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-direct-children) like:

```html
<tr className="*:px-6 *:py-4 *:whitespace-nowrap *:text-small">
```

**Reply by JaimeenMakavana** [reply]
thanks !
